README.md

```diff
@@ -23,4 +23,71 @@ A build using an optical switch can be found at http://www.thingiverse.com/thing
 Note: Needs RPi.GPIO version greater than 0.6.0 to allow access to GPIO for non root and `chmod a+rw /dev/gpiomem`.
 This requires a fairly up to date system.
 
-WARNING: I am **not responsible** for any failed prints. Use at your own risk.
\ No newline at end of file
+WARNING: I am **not responsible** for any failed prints. Use at your own risk.
+
+=== How to install ===
+
+For some older Octopi installs, the plugin won't work before you make sure you have everything set up to date. Here are the steps to ensure you have everyting in place for the plugin.
+ 0- you can check your system version with 
+- `uname -a 
```

Comment:
@jonaslorander you're right, this should be corrected
@Actpohomoc thanks, happy that it helped you!